### PR TITLE
ordering/finished

### PR DESCRIPTION
**Pull request by RafaelNagiyev**

### CODE REVIEW

File: src/components/OrderingPage/OrderingPart.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+export const OrderingPart = () => {
+  const [count, setCount] = useState(1);
+  const incrementCount = () => {
+    setCount(count + 1);
+  };
+
+  const [{ items }, setItems] = useState({ items: [<></>] });
+  const addItem = () => {
+    items.push(
```
Comment (Negodyaj) [inline src/components/OrderingPage/OrderingPart.tsx]: you do it slightly wrong here)
you need to create a component for the html below and you have to store only data (an array of objects) in the component's state

File: src/components/OrderingPage/OrderingPart.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            </div>
+            {count > 0 && (
+              <div className="close-order">
+                <button className="close-order-button">
+                  <img src={x} alt="" />
+                </button>
+              </div>
+            )}
+          </div>
+          <OrderingForm />
```
Comment (Negodyaj) [inline src/components/OrderingPage/OrderingPart.tsx]: are you sure that this component must be here?
in this case you will have several order forms on the page... for me, that's strange)

File: src/components/OrderingPage/OrderingPart.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+              {count > 1 && (
+                <div className="close-order">
+                  <button className="close-order-button">
+                    <img src={x} alt="" />
+                  </button>
+                </div>
+              )}
+            </div>
+            <OrderingForm />
+          </div>
```
Comment (Negodyaj) [inline src/components/OrderingPage/OrderingPart.tsx]: and again... the same html here...

File: src/components/OrderingPage/OrderingPart.tsx (hunk 2 above)
Comment (Negodyaj) [reply src/components/OrderingPage/OrderingPart.tsx]: you should do something with this code in this case))

Comment (RafaelNagiyev) [reply src/components/OrderingPage/OrderingPart.tsx]: but it is working) I dont understand what is wrong with it

**Pull request merged**